core/templates/dev/head/app.js

```diff
@@ -617,6 +617,7 @@ oppia.factory('extensionTagAssemblerService', [
           $filter('camelCaseToHyphens')(caSpecName) + '-with-value',
           oppiaHtmlEscaper.objToEscapedJson(caSpecValue));
       }
+      element.attr('on-submit', 'submitAnswer(answer, rulesService);');
```

Comment:
You'd need to do this somewhere higher up the stack -- this service is used for both interactions and gadgets IIRC. The latter doesn't have an answer submission mechanism.

Look in oppiaExplorationHtmlFormatterService.getInteractionHtml() perhaps? (in core/templates/dev/head/services)